site/src/pages/WorkspacePage/Workspace.tsx

```diff
@@ -265,7 +265,7 @@ export const Workspace: FC<React.PropsWithChildren<WorkspaceProps>> = ({
               />
             </Cond>
             <Cond>
-              {/* <ImpendingDeletionBanner/> determines its own visibility */}
```

Comment:
the classic "comment doesn't get updated even though it was *right* next to the code 😂